### DISCUSSION TITLE
README

### DISCUSSION THREAD

**Discussion by Pcontreras98** [community/community#127298] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I created a readme on my profile to pin it but it won't let me, any help?
![Pin](https://github.com/community/community/assets/85205462/1e6cfe7d-9ffd-40ae-9c59-9510b85bd573)

**Comment by davevad93** [comment, marked as answer]
Hi @Pcontreras98 ,

By clicking on `Customize your pins` you should be able to choose up to 6 repos to pin on your profile. One more thing, I've noticed that you don't actually have a readme file but only a repository called `readme` to make a profile's `README` you have to create a repository called `Pcontreras98`.

**Reply by Pcontreras98** [reply]
thank u so much, i fixed it.

**Reply by davevad93** [reply]
You're welcome @Pcontreras98 have a nice day!

**Comment by ebndev** [comment]
Hi @Pcontreras98, thanks for being a part of the GitHub Community!

If you need more inspiration for your README, feel free to check out our Discussion [Showcase Your GitHub Profile 🌟 ](https://github.com/orgs/community/discussions/121833). There are a lot of different things you can add but keeping it simple is great too if that's what you decide to do 😄
